eZ/Bundle/PlatformInstallerBundle/src/Installer/CoreInstaller.php

```diff
@@ -67,16 +66,9 @@ public function importSchema()
         $progressBar = new ProgressBar($this->output);
         $progressBar->start($queriesCount);
 
-        try {
-            $this->db->beginTransaction();
-            foreach ($queries as $query) {
-                $this->db->exec($query);
-                $progressBar->advance(1);
-            }
-            $this->db->commit();
-        } catch (DBALException $e) {
-            $this->db->rollBack();
-            throw $e;
+        foreach ($queries as $query) {
+            $this->db->exec($query);
+            $progressBar->advance(1);
```

Comment:
Why do you need to get rid of this? If there's no transaction, a transaction is per commit which means that in case of any failure, partial installation will be persisted.

Reply:
MySQL driver received a change in PHP 8.0 that makes attempting to commit a transaction when there isn't any an error. Because DDL operations do not allow transactions in MySQL this means that transaction is implicitly committed. This results in an error.

A similar solution was made for doctrine/migrations, where it was suggested that mysql Migrations should explicitly be made outside of transaction. See https://github.com/doctrine/migrations/issues/1104.

Reply:
As of PHP 8, `PDO::commit()` and `rollBack()` will throw a `There is no active transaction` PDOException if the transaction had been autocommitted, Previously they would just fail silently.

Reply:
Fair enough, can't do much about that, +1